src/slot_machine.c

```diff
@@ -6727,15 +6727,12 @@ static const struct SpriteSheet gSlotMachineSpriteSheets[] =
     { .data = gSlotMachineNumber7Tiles, .size = 0x40, .tag = 14 },
     { .data = gSlotMachineNumber8Tiles, .size = 0x40, .tag = 15 },
     { .data = gSlotMachineNumber9Tiles, .size = 0x40, .tag = 16 },
-};
-
-static const u8 sUnused1[][8] = 
-{
-    {0, 0, 0, 0, 0, 2, 18},
-    {0, 0, 0, 0, 0, 2, 19}, 
-    {0, 0, 0, 0, 0, 3, 20}, 
-    {0, 0, 0, 0, 0, 3, 21}, 
-    {0, 0, 0, 0, 0, 0,  0}
+    // the data for these sheets is determined at runtime
+    { .data = NULL, .size = 0x200, .tag = 18 },
+    { .data = NULL, .size = 0x200, .tag = 19 },
+    { .data = NULL, .size = 0x300, .tag = 20 },
+    { .data = NULL, .size = 0x300, .tag = 21 },
+    { .data = NULL, .size = 0x0, .tag = 0 },
```

Comment:
Terminator element does not need explicit field initializers.

Reply:
Well, yes. I mainly wrote it like that due to readability.
Should I remove the last line?